ProcessQueue/function.json

```diff
@@ -5,7 +5,7 @@
       "name": "msg",
       "type": "queueTrigger",
       "direction": "in",
-      "queueName": "scanreports",
+      "queueName": "%SCAN_REPORT_QUEUE_NAME%",
```

Comment:
This var (%SCAN_REPORT_QUEUE_NAME%) is being pulled from local.settings.json (not tracked in Git for security reasons)